AsyncDisplayKit/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -583,6 +584,12 @@ - (void)setBackgroundColor:(UIColor *)newBackgroundColor
   if (shouldApply) {
     CGColorRef oldBackgroundCGColor = _layer.backgroundColor;
     _layer.backgroundColor = newBackgroundCGColor;
+    
+    BOOL specialPropertiesHandling = ASDisplayNodeNeedsSpecialPropertiesSettingHandlingForFlags(_flags);
+    if (specialPropertiesHandling) {
+        _view.backgroundColor = newBackgroundColor;
```

Comment:
I think in this case we should probably not set the backgroundColor on the layer.  Setting it on the view should always do that, unless perhaps the view class intentionally wants to do something else with it (perhaps it needs to redirect it to be set on a sublayer or such).

So let's fork the code inside shouldApply to the view and the layer case.